### DISCUSSION TITLE
How to plot a list of texture components {hkl} <uvw> in one pole figure ?

### DISCUSSION THREAD

**Discussion by jucego333** [mtex-toolbox/mtex#1043] [Ask Anything]
Hello dear colleagues

I am studying the slip systems {332}<113> and would like to plot all the variants into a (332) pole figure, such that every variant has its own color or mark. Here there is the list of Miller Index:
(3 3 2)[ 1 1-3]
(3 2 3)[ 1-3 1]
(2 3 3)[-3 1 1]
(3 -3 2)[1 -1 -3]
(3 -2 3)[1  3  1]
(2 -3 3)[-3 -1 1]
(-3 3 2)[-1 1 -3]
(-3 2 3)[-1 -3 1]
(-2 3 3)[3 1 1]
(3 3 -2)[1 1 3]
(3 2 -3)[1 -3 -1]
(2 3 -3)[-3 1 -1]
However, I have not obtained the pole figure successfully.  This the code that I used:

ori = orientation.byMiller([3 3 2],[1 1 -3],cs,ss)  %%%   plane , direction
d = Miller(3,3,2,cs,'uvw');  % pole plane
plot(ori,'MarkerFaceColor',[0 0.5 0],'DisplayName','(3 3 2)<1 1-3>')
hold on
ori1 = orientation.byMiller([3 2 3],[1 -3 1],cs,ss)  %%%   plane , direction
d1 = Miller(3,2,3,cs,'uvw');  % pole plane
plot(ori1,'MarkerFaceColor',[1 0 0],'DisplayName','(3 2 3)<1 -3 1>')

In addition,  I found a similar discussion here https://github.com/mtex-toolbox/mtex/discussions/724 ,  but they used EBSD data instead of Miller index, and also the use pdf prompt is not clear.  I also reviewed carefully the documentation, but no progress so far.  Could anybody give me a guideline? Thanks in advance. 

![image](https://user-images.githubusercontent.com/78469467/118675388-25abfe80-b82d-11eb-88b9-90a4049c64f8.png)

**Comment by ralfHielscher** [comment]
I do not understand what you want to do. Do you want display different variants as they appear in an parent / child orientation relationship? What ist the relation between slip systems and orientations?

Please explain more in detail.

Ralf.

**Reply by jucego333** [reply]
Dear Dr @ralfHielscher  Thank you for the time taken in this question. Well, I am currently working with a beta-titanium alloy and I want to analyze the twinning in the system {332}<113>, as they did here https://doi.org/10.1016/j.actamat.2018.10.003.  (Picture attached) by using EBSD and pole figures. Then, I would like to know if it is possible to display the different twining variants (24)  into a pole figure such that I can identify the existing variants in my EBSD data.  I would deeply appreciate your comments. Thanks

![image](https://user-images.githubusercontent.com/78469467/119084252-07fcb600-ba34-11eb-8a4d-45fe4afb245f.png)
